node/resolvers/search/index.ts

```diff
@@ -654,19 +658,23 @@ export const queries = {
 
     const [count, page] = getProductsCountAndPage(from, to)
 
-    const biggyArgs = {
+    const workspaceSearchParams = await getWorkspaceSearchParamsFromStorage(ctx)
+
+    const biggyArgs : SearchResultArgs = {
       page,
       count,
       fuzzy,
       operator,
       searchState,
       attributePath: buildAttributePath(selectedFacets),
       query: fullText,
+      fullText,
```

Comment:
are we using this?

Reply:
It's not nullable, so I'm not quite sure how it was missing before.

```
interface SearchResultArgs {
  attributePath: string
  query?: string
  page?: number
  count?: number
  sort?: string
  operator?: string
  fuzzy?: string
  leap?: boolean
  tradePolicy?: number
  segment?: SegmentData
  indexingType?: IndexingType
  fullText: string
  searchState?: string
  sellers?: RegionSeller[]
  hideUnavailableItems?: boolean | null
  removeHiddenFacets?: boolean | null
  options?: Options
  workspaceSearchParams?: object
}
```

Reply:
`biggySearch.productSearch` doesn't use this parameter, we've already passed its value in `query: fullText`. I think it exists to be used in `banners` but as it's not used in `productSearch` we don't need to pass it here